controller/stats.controller.js

```diff
@@ -69,7 +71,26 @@ const getUsersStats = async (req, res) => {
     }
 }
 
+const getTestsPerMonth = async (req, res) => {
+
+    const accountID = req.params.accountId;
+    if (!mongoose.isValidObjectId(accountID)) throw new ValidationError.MissingPropertyError("account ID");
+    Experiment.aggregate([
```

Comment:
Very nice logic in this function and a very elegant way of coding
though this should not be here, this should be in the Experiment Repository as mentioned above